docs/devices/SNZB-02.md

```diff
@@ -15,6 +15,7 @@ description: "Integrate your SONOFF SNZB-02 via Zigbee2MQTT with whatever smart
 | Exposes | battery, temperature, humidity, voltage, linkquality |
 | Picture | ![SONOFF SNZB-02](../images/devices/SNZB-02.jpg) |
 | White-label | eWeLink RHK08 |
+| Battery type | CR2450 |
```

Comment:
This part is generated (and thus it will automatically be removed on regeneration), please move to notes.